sphinx/source/docs/dev_guide/creating_issues.rst

```diff
@@ -0,0 +1,143 @@
+the GitHub repository at `<https://github.com/bokeh/bokeh/issues>`_.
+
+.. _devguide_issues_before:
+
+Before opening an issue
+-----------------------
+
+As a first step, you should always check
+`Bokeh's GitHub issue tracker <issue tracker_>`_ and the
+`Bokeh Discourse`_ for similar or identical issues.
+
+It is also possible that the issue you want to report has already been resolved
+in a newer version of Bokeh. Update Bokeh to the most recent stable version
+available with `conda`_ or `pip`_ and try to reproduce the issue.
+
+You can format any text you enter when filing an issue. To learn more about
+your formatting options, see `Writing on GitHub`_ in the `GitHub documentation`.
+
+.. _devguide_issues_kinds:
+
+Kinds of issues
+---------------
+
+Bokeh's issue tracker accepts two kinds of issues:
+
+Bug report
+    Please use this kind of issue to report anything that does not work as it
+    should. This can be an unexpected error, a fault in the code, or a
+    discrepancy between the documentation and the actual behavior of the code,
+    for example.
+
+Feature request
+    Please use this kind of issue to propose a new feature or anything else that
+    expands Bokeh's current capabilities.
+
+When you `create a new issue`_, you can choose between creating these two types
+of issues. When creating either type of issue, GitHub will present you with a
+template. Please follow the instructions in those templates closely and provide
+all necessary information.
+
+.. note::
+    The issue tracker is not the place for general support. For questions and
+    technical assistance, visit the `Bokeh Discourse`_. When you are unsure,
+    post a question to the Discourse first before opening an issue.
+
+.. _devguide_issues_kinds_bugs:
+
+Reporting bugs
+~~~~~~~~~~~~~~
+
+To `file a new bug report`_, you should have the following information ready.
+This will help with resolving your issue as quickly as possible:
+
+Software version info
+    To find the version of packages such as Bokeh, Python, IPython, and Node.js,
+    use the following command in the environment where you have Bokeh installed:
+
+    .. code-block:: sh
+
+        bokeh info
+
+    This provides you with a list of the versions of relevant software packages.
+    Copy and paste this information into your bug report.
+
+    Additionally, you should also provide information about your web browser's
+    version, the version of your operating system, and the version of any
+    other packages relevant to the bug you are reporting.
+
+Description of expected behavior and the observed behavior
+    Describe the behavior you would have expected and the behavior you observed
+    instead.
+
+Complete, minimal, self-contained example code that reproduces the issue
+    Please provide a code example that follows the
+    `definition of a minimal, reproducible example at stackoverflow.com`_.
+    For more details, see `Craft Minimal Bug Reports at matthewrocklin.com`_.
```

Comment:
I don't know if there is a good place to fit it in but FWIW it is 1000% better to get MREs as simple scripts and not code to run in notebook, unless the issue is very specifically to do with the notebook in particular. Also screenshots of code are always *very discouraged*